scripts/ci-scripts/test-android.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+SRCROOT=$(cd "$(dirname "$0")/.." && pwd)
+PACKAGER_OUT="$SRCROOT/packager_out.txt"
+LOGCAT_OUT="$SRCROOT/logcat_out.txt"
+RUN_STITCH_IN_FORGROUND=""
+PATH="/opt/android-sdk-linux/platform-tools:$PATH"
+XCPRETTY=$(which xcpretty || true)
+
+start_packager() {
+  rm -r $TMPDIR/react-* || true
+  watchman watch-del-all || true
+  ./node_modules/react-native/scripts/packager.sh --reset-cache | tee "$PACKAGER_OUT" &
+
+  while :; do
+    if grep -Fxq "Loading dependency graph, done." "$PACKAGER_OUT"; then
+      break
+    else
+      echo "Waiting for packager."
+      sleep 2
+    fi
+  done
+}
+
+# pack realm package manually since install-local does not allow passing --ignore-scripts
+echo "manually packing realm package"
+npm pack .
+rm -rf realm.tgz
+mv realm-*.*.*.tgz realm.tgz
+
+echo "manually packing realm tests package"
+pushd tests/js
+npm pack .
+rm -rf realm-tests.tgz
+mv realm-tests-*.*.*.tgz realm-tests.tgz
+popd
+
+pushd tests/react-test-app
+echo "installing react-test-app dependencies"
+npm ci --no-optional
+
+echo "installing manually packed realm package"
+npm install --save-optional  --ignore-scripts ../../realm.tgz
+
+echo "installing manually packed realm tests package"
+npm install --save-optional ../js/realm-tests.tgz
+
```

Comment:
Could all of these be combined into a single `npm install` invocation to save some time?